dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/scalaUtils/PythonEnum.scala

```diff
@@ -0,0 +1,11 @@
+package me.shadaj.scalapy.tensorflow.api.scalaUtils
+
+import scala.language.implicitConversions
+
+class PythonEnum(private[api] val v: String)
+
+object PythonEnum {
+
+  implicit def enum2String: Conversion[PythonEnum, String] = _.v
```

Comment:
Nit: can this be `val`? same for other Conversions across the codebase

Reply:
But why? those are methods